nautobot/extras/homepage.py

```diff
@@ -0,0 +1,136 @@
+from nautobot.core.apps import HomePageColumn, HomePageItem, HomePagePanel
+from nautobot.extras.choices import JobResultStatusChoices
+from nautobot.extras.models import GitRepository, JobResult, ObjectChange
+
+
+def get_job_results(_):
```

Comment:
Why does this take an argument? Is there a way to describe this a little better, or perhaps rename the argument to describe why it is not used in a simple docstring?